src/rgw/rgw_sal_daos.cc

```diff
@@ -862,10 +850,7 @@ int DaosStore::initialize(void) {
   uuid_copy(pool, pool_info.pi_uuid);
 
   // Connect to metadata container
-  // TODO use dfs_connect
-  // Attempt to create
-  ret = dfs_cont_create_with_label(poh, METADATA_BUCKET, nullptr, nullptr,
-                                   &meta_coh, &meta_dfs);
```

Comment:
Are there other instances of `dfs_cont_create_with_label` in the code? How about `daos_cont_open` and `daos_cont_close`?